### ISSUE TITLE
Reduce file size for report

### ISSUE THREAD

**Issue by pinin4fjords** [nf-core/differentialabundance#106]
### Description of feature

(sorry @WackerO , you were right!)

I'm noticing my browser having trouble coping with reports (even if they are only 30Mb or so). 

We may have to do some sensible truncation of the d/e tables etc, and take steps to reduce the size of the HTML associated with different plots (maybe some rounding is in order).

**Comment by WackerO** [comment]
yeah, I've had the same issue...So far my browser can handle the report, but it does take a noticeable time to load. I'm wondering also if we could make some parts of the report optional. I'm thinking maybe by default don't add the methods section; the parameters could instead just be written into a tsv file. Also, what do you think about by default only having the 2D PCA plots and adding the 3D plot only if the user sets this with some param?

**Comment by pinin4fjords** [comment]
The method section won't be having a major impact, neither will the PCA (the number of points, and therefore the amount of embedded data is small). I would put money on the differential expression tables (not currently truncated) and many-point plots such as the volcano plots as the major culprits.

**Comment by alanzos** [comment]
Yup, I have also suffered the long loading times and sometimes it freezes.

A sub-optimal solution for me has been to limit the number of contrasts to 3-4, and generate several reports.

An ideal solution would be to have a parameter that changes the interactive plots, to static ones, like outlier detection plots, density plots, PCAs, and volcanos (as @pinin4fjords mentioned). This should significantly reduce the size of the report.

**Comment by atrigila** [comment]
This should have been solved by: https://github.com/nf-core/differentialabundance/pull/638 and https://github.com/nf-core/differentialabundance/pull/748
